sdk/typescript/test/unit/cryptography/ed25519-publickey.test.ts

```diff
@@ -83,13 +70,6 @@ describe('Ed25519PublicKey', () => {
     expect(new Ed25519PublicKey(key.toBytes()).equals(key)).toBe(true);
   });
 
-  it('toSuiAddress', () => {
```

Comment:
Curious why this test was removed?

Reply:
https://github.com/MystenLabs/sui/pull/9262/files/312c2be35eb902df8dfc90fbf8ca2e2d8ccd14ad#diff-c4c301721c06ce4af4e64ffa7d26b33ddff61631a71a574abcf532a74c583d79R73 

there are a bunch of tests here already for the same purpose